### ISSUE TITLE
getSegmentData - Orientation around 0 of spectral values

### ISSUE THREAD

**Issue by agataelia** [eMapR/LT-GEE#10]
Hi @jdbcode , 

I am using the getSegmentData(lt, index, delta, right) function with NBR index, and having some troubles in understanding the orientation around zero of the output segments values.

I tried running:
`var segInfo = ltgee.getSegmentData(lt, index, 'all')`
and got this result for pixel A:
![image](https://user-images.githubusercontent.com/62068145/122379491-343e2080-cf67-11eb-8da7-2e8d1eb1f4c7.png)

Then I tried running:
`var segInfo = ltgee.getSegmentData(lt, index, 'all', true)`
and got this result for pixel A:
![image](https://user-images.githubusercontent.com/62068145/122379543-3f914c00-cf67-11eb-9516-53c6fbe36579.png)

For final comparison I tried running:
`var segInfo = ltgee.getSegmentData(lt, index, 'loss')`
and got this result for pixel A:
![image](https://user-images.githubusercontent.com/62068145/122379586-48821d80-cf67-11eb-8b96-438f77f730b1.png)

So what I am wondering is whether the ‘right’ parameter set to ‘true’ is not doing its job here (flipping the values to their natural orientation around zero) or am I not using it correctly?

Thank you!
Best,
Agata

**Comment by agataelia** [comment]
Hello! Me again, but I am really using your module and indeed thanks for creating such an interesting implementation and useful APIs. 

Still in the getSegmentData function, I noticed that you increment of 1 every start year:

`return ee.Image.cat([startYear.add(1), endYear, startVal, endVal, mag, dur, rate, dsnr]`)

So you basically assign every vertex to the segment at the left of the vertex. May I ask why did you make this choice? Is it in order not to have the endYear of a segment coincide with the startYear of the subsequent segment? Also the duration it is still returned as endYear - startYear so I found it a bit hard to wrap my head around it at first, so I wanted to ask you whether is there a computational reason behind this.

Thank you!

**Comment by jdbcode** [comment]
Please separate these questions into different issues. I'll get back to you when I'm off vacation.

**Comment by jdbcode** [comment]
Yes, this seems to be a bug.

Only default `false` is working. The logic to handle an input boolean is not WAI. Here is an idea for how it should be handled:

``` 
var getSegmentData = function(options){
  var _options = {right: false};
  if (typeof options == 'boolean') {
    _options.right = options;
  }
  print(_options.right);
};

getSegmentData(true);
```

Before making the change, will need to determine if this will break anything calling `getSegmentData`, like `getChangeMap`.

**Comment by jdbcode** [comment]
Script to test the changes: https://code.earthengine.google.com/0ef186eed77e8ff910b82ae1b4a64c41

**Comment by jdbcode** [comment]
Closed by #15
